core/src/main/java/org/infinispan/stream/impl/DistributedCacheStream.java

```diff
@@ -72,11 +78,12 @@
 import org.infinispan.stream.impl.intops.object.PeekOperation;
 import org.infinispan.stream.impl.termop.object.ForEachBiOperation;
 import org.infinispan.stream.impl.termop.object.ForEachOperation;
-import org.infinispan.stream.impl.termop.object.NoMapIteratorOperation;
-import org.infinispan.util.CloseableSuppliedIterator;
 import org.infinispan.util.RangeSet;
```

Comment:
should use `org.infinispan.commons.util.RangeSet`

Reply:
Sure I can update all of these.